datadog/README.md

```diff
@@ -0,0 +1,5 @@
+# Datadog Support
```

Comment:
Can you add that this requires APM (and metrics I guess even if it's implied :sweat_smile:) products to be enabled?
Maybe even giving an order of magnitude of expected custom metrics so there's no billing surprise for users?

Reply:
done for top level but order of magnitude estimate feels like a bad idea (environment dependent, what if we get it worng etc...)